.travis.yml

```diff
@@ -26,8 +26,12 @@ install:
   - if [[ $TRAVIS_PYTHON_VERSION == '3.3' ]]; then conda create --yes -q -n pyenv mkl python=3.3 numpy scipy nose pyparsing pip flake8 six pep8 pyflakes; fi
   - if [[ $TRAVIS_PYTHON_VERSION == '3.4' ]]; then conda create --yes -q -n pyenv mkl python=3.4 numpy scipy nose pyparsing pip flake8 six pep8 pyflakes; fi
   - source activate pyenv
+  - curl -O https://raw.githubusercontent.com/ASalvail/blocks/master/req-travis-conda.txt
```

Comment:
Can you try using this one:
`https://raw.githubusercontent.com/mila-udem/blocks/master/req-travis-conda.txt`